### ISSUE TITLE
Can't change the cursor position by clicking the EditText or keyboard

### ISSUE THREAD

**Issue by yisizhu520** [Yalantis/ToDoList#1]
The "Plus" animation is delightful and It is smooth to replace the EditText's cursor with the custom cursor.  But the problem I find is that I can't change the cursor position by clicking the EditText or soft keyboard, and the actual input position has changed. I think it confuses users since input is important for a to-do list.

**Comment by igalata** [comment]
@yisizhu520 Hi! Thank you for this issue. Yes, it's really important to be able to manually change the cursor position. I'll implement it very soon

**Comment by igalata** [comment]
Hello, @yisizhu520
The 1.0.1 version is available, please check it. Now you can move the cursor by clicking the EditText.
Review the changelog for more details.

**Comment by yisizhu520** [comment]
Yeah, the change is what I exactly need and better than my idea. I set the `textCursorDrawable` attribute for the EditText:
```
android:textCursorDrawable="@drawable/ic_cursor_drawable"
```
And remove the mask view. It works but the cursor animation is not as smooth as origin custom cursor. Because the height of cursor depends on the text size of EditText and the left padding need be adjusted.     
Learn a lot from the library. Thanks!

**Comment by igalata** [comment]
Hi @yisizhu520! Sorry for the delay in responding. I'll fix the issue very soon. Thanks a lot for your attention to my library

**Comment by VPOGOR** [comment]
x
